tensorflow/core/util/mkl_util.h

```diff
@@ -135,6 +135,17 @@ inline void execute_primitives(
   }
 }
 
+bool inline AreWeightsFrozen() {
```

Comment:
Should the implementation be in .cc file instead? Otherwise, `mkl_conv_ops` and `mkl_matmul_op_fused` will have two separate copies of the function and will read the environment variable separately. If this gets use in more files, `ReadBoolFromEnvVar` will be called even more redundantly.

Reply:
@penpornk only these two primitives/files deal with weights/filters and caching them. So we dont expect anyother files to use this functionality. If you are okay we can leave it as is. what do you suggest?

Reply:
Let's put in in `mkl_util.cc` anyway. Thank you very much!

Reply:
@penpornk Made the changes. Sorry for the delay. We had to go through some CI since it touches build.